traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/CatalogItemDetailsFragment.java

```diff
@@ -178,6 +186,10 @@ public void onWishlistAddError(Error error) {
     @Override
     public void onWishlistRemoveSuccess(Product item, CatalogItemDetails itemDetails) {
         catalogItemDetails = itemDetails;
+        Activity activity = getActivity();
+        if (activity instanceof CatalogItemDetailsActivity) {
```

Comment:
same here. this shows something wrong in the way things are done

Reply:
updated